config/mysqlpersister/mysqlpersister.go

```diff
@@ -0,0 +1,210 @@
+package mysqlpersister
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+	"sync"
+	"time"
+
+	sq "github.com/Masterminds/squirrel"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/golang/protobuf/proto"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/square/quotaservice/config"
+	"github.com/square/quotaservice/logging"
+	qsc "github.com/square/quotaservice/protos/config"
+)
+
+type mysqlPersister struct {
+	latestVersion int
+	db            *sqlx.DB
+	m             *sync.RWMutex
+
+	watcher  chan struct{}
+	shutdown chan struct{}
+
+	activeFetchers *sync.WaitGroup
+
+	configs map[int]*qsc.ServiceConfig
+}
+
+type configRow struct {
+	Version int    `db:"Version"`
+	Config  string `db:"Config"`
+}
+
+func New(dbUser, dbPass, dbHost string, dbPort int, dbName string, pollingInterval time.Duration) (config.ConfigPersister, error) {
+	db, err := sqlx.Open("mysql",
+		fmt.Sprintf("%s:%s@(%s:%v)/%s",
+			dbUser,
+			dbPass,
+			dbHost,
+			dbPort,
+			dbName))
+
```

Comment:
nit: I'd get rid of this new line

Reply:
Wow, the first actual nit I've seen 😉 Pushed